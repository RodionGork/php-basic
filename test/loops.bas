x=10
repeat:
105 print x
x = x+20
if x < 130 then goto repeat
print "done"
if x <= 210 then goto 105
goto retreat
