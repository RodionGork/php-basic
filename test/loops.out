10
30
50
70
90
110
done
130
done
150
done
170
done
190
done
210
done
Runtime error (line #8): No such label: RETREAT
